controller/search_blackbox_test.go

```diff
@@ -1050,6 +1050,175 @@ func (s *searchControllerTestSuite) TestSearchByJoinedData() {
 	})
 }
 
+// TestSearchWorkItemsWithChildIterationsOption verifies the Included list of parents
```

Comment:
Can you please fix the comment?

Reply:
Fixed in 387c959b310b510968d8bdf62079a5e75b11e8bd